easybuild/easyconfigs/s/SAMtools/SAMtools-1.7-foss-2018a-HTSlib-1.7.eb

```diff
@@ -0,0 +1,57 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Robert Schmidt <roschmidt@ohri.ca>, Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+#
+# Modified for version 1.4 by: Kurt Lust, UAntwerpen
+#
+##
+
+easyblock = 'ConfigureMake'
```

Comment:
@tutufan We have a custom easyblock for `SAMtools`, so we shouldn't use `ConfigureMake` here?